src/test/java/org/cactoos/http/HtTimedWireTest.java

```diff
@@ -76,24 +84,145 @@ public void worksFine() throws Exception {
+        ), new AllOf(
+            new TimeoutExceptionMatcher(),
+            new TypeSafeDiagnosingMatcher<Scalar<Input>>() {
+                @Override
+                protected boolean matchesSafely(
+                    final Scalar<Input> item,
+                    final Description description) {
+                    final long failed = System.currentTimeMillis();
+                    return failed - current < sleep;
+                }
+
+                @Override
+                @SuppressWarnings("PMD.UncommentedEmptyMethodBody")
+                public void describeTo(final Description description) {
+                }
+            }));
+    }
+
+    /**
+     * Class that matches that {@link java.util.concurrent.TimeoutException}
+     * is thrown.
+     * @todo #87:60m For now we can't use
+     *  {@link org.llorllale.cactoos.matchers.Throws}
+     *  to match exception message because message in the exception is null.
+     *  I've tried to update to the new version of matchers and
+     *  use constructor of Throws with {@link org.hamcrest.Matcher},
+     *  but it leads update the cactoos
+     *  and after that I can't build a project.
+     *  So needs to update dependencies, fix conflicts and replace
+     *  this class with {@link org.llorllale.cactoos.matchers.Throws}.
+     *  And needs try to remove {@link java.lang.SuppressWarnings} in
+     *  {@link org.cactoos.http.HtTimedWireTest}.
+     */
+    private final class TimeoutExceptionMatcher extends
+        TypeSafeDiagnosingMatcher<Scalar<Input>> {
+
+        @Override
+        @SuppressWarnings("PMD.AvoidCatchingGenericException")
+        public boolean matchesSafely(
+            final Scalar<Input> item,
+            final Description description) {
+            // @checkstyle IllegalCatchCheck (10 lines)
+            boolean matches;
+            try {
+                item.value();
+                matches = false;
+            } catch (final TimeoutException exception) {
+                matches = exception.getMessage() == null;
+            } catch (final Exception exception) {
+                matches = false;
+            }
+            return matches;
+        }
+
+        @Override
+        public void describeTo(final Description description) {
+            //Not used
+        }
+    }
+
+    /**
+     * Class that matches that use
+     * all of {@link org.cactoos.http.HtTimedWireTest.AllOf#matchers}.
+     * @todo #87:30m For a some reason org.hamcrest.core.AllOf matcher with
+     *  org.cactoos.iterable.IterableOf is not working (compilation error with
+     *  unchecked generic array creation for varargs parameter).
+     *  Needs to replace this with org.hamcrest.core.AllOf
+     *  and deal with compilation error.
+     *  And needs try to remove {@link java.lang.SuppressWarnings} in
+     *  {@link org.cactoos.http.HtTimedWireTest}.
+     */
+    private final class AllOf extends TypeSafeDiagnosingMatcher<Scalar<Input>> {
```

Comment:
@aivinog1 why not use `AllOf` from hamcrest?

Reply:
Because it requires Iterable, but when I create Iterable with a generic anonymous class, then it fails. I'll fix it (replace generic anonymous class to an inner class ).

Reply:
And I'll add a todo that you posted here: https://github.com/yegor256/cactoos-http/pull/92#discussion_r299099594

Reply:
> Because it requires Iterable, but when I create Iterable with a generic anonymous class, then it fails. I'll fix it (replace generic anonymous class to an inner class ).

A quick fix cannot help me, I've added a todo.